dianna/dashboard/Home.py

```diff
@@ -1,6 +1,8 @@
 import streamlit as st
 from _shared import add_sidebar_logo
 from _shared import data_directory
+from streamlit_option_menu import option_menu
+import importlib
```

Comment:
```suggestion
import importlib
import streamlit as st
from _shared import add_sidebar_logo
from _shared import data_directory
from streamlit_option_menu import option_menu
```

Reply:
Here I just moved the `importlib` to the top to make `isort` happy. 

Can you please add `streamlit_option_menu` to `dashboard` in [setup.cfg](https://github.com/dianna-ai/dianna/blob/430be09d51c26cfc5fc7ed9b87126cb7965a544b/setup.cfg#L90)?